Bottom: 0b63be38b528c22cca28bfcaf35309a00eee797b
Top:    668e4efd2c42c95930cf5b6b0a761c744d84751d
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 19:04:28 -0500

Refresh of jds-lirc-core

---

diff --git a/drivers/input/ir/Makefile b/drivers/input/ir/Makefile
index 2ccdda3..6acb665 100644
--- a/drivers/input/ir/Makefile
+++ b/drivers/input/ir/Makefile
@@ -4,5 +4,5 @@
 # Each configuration option enables a list of files.
 
 obj-$(CONFIG_INPUT_IR)		+= ir.o
-ir-objs := ir-core.o ir-configfs.o
+ir-objs := ir-core.o
 
diff --git a/drivers/input/ir/ir-core.c b/drivers/input/ir/ir-core.c
index 8e08f3f..1eff985 100644
--- a/drivers/input/ir/ir-core.c
+++ b/drivers/input/ir/ir-core.c
@@ -10,6 +10,12 @@
 
 #include "ir.h"
 
+void input_ir_translate(struct input_dev *dev, int protocol, int device, int command)
+{
+
+}
+
+
 static int encode_sony(struct ir_device *ir, struct ir_command *command)
 {
 	/* Sony SIRC IR code */
@@ -664,15 +670,11 @@ EXPORT_SYMBOL_GPL(input_ir_destroy);
 
 static int __init input_ir_init(void)
 {
-	config_group_init(&input_ir_remotes.su_group);
-	mutex_init(&input_ir_remotes.su_mutex);
-
-	return configfs_register_subsystem(&input_ir_remotes);
+	return 0;
 }
 module_init(input_ir_init);
 
 static void __exit input_ir_exit(void)
 {
-	configfs_unregister_subsystem(&input_ir_remotes);
 }
 module_exit(input_ir_exit);
